model/project_parser_test.go

```diff
@@ -893,7 +893,6 @@ buildvariants:
 	_, err = LoadProjectInto(ctx, []byte(yamlWithVariables), opts, "example_project", &proj)
 	require.NoError(t, err)
 
-	fmt.Println("MY TESTS ARE STARTING")
```

Comment:
👀  lol

Reply:
yeah who in the world would commit test code like this *hyperventilates*